code/game/objects/structures/vehicles/wheelchair.dm

```diff
@@ -152,3 +152,105 @@
 	..()
 
 	update_mob() //Update the rest
+
+/obj/structure/bed/chair/vehicle/wheelchair/motorized
+	name = "motorized wheelchair"
+	nick = "cripplin' revenge"
+	desc = "A chair with fitted wheels which is powered by an internal cell. It propels itself without the need for hands as long as it is charged."
+	var/maintenance = 0
+	var/motor_delay = 1
+	var/obj/item/weapon/cell/internal_battery = null
+
+
+/obj/structure/bed/chair/vehicle/wheelchair/motorized/New()
+	..()
+	internal_battery = new /obj/item/weapon/cell/high(src)
+
+/obj/structure/bed/chair/vehicle/wheelchair/motorized/examine(mob/user)
+	..()
+	if(internal_battery)
+		var/calc = round((internal_battery.charge / internal_battery.maxcharge) * 100,1)
+		user << "<span class='info'>The battery meter reads: [calc]%</span>"
+	else
+		user << "<span class='warning'>The 'check battery' light is blinking.</span>"
+
+/obj/structure/bed/chair/vehicle/wheelchair/motorized/Move()
+	..()
+	internal_battery.use(2) //Example use: 100 charge to get from the cargo desk to medbay side entrance
+
+/obj/structure/bed/chair/vehicle/wheelchair/motorized/getMovementDelay()
+	if(internal_battery && internal_battery.charge)
+		return motor_delay
+	else
+		return (..() * 2) //It's not designed to move this way!
+
+/obj/structure/bed/chair/vehicle/wheelchair/motorized/check_key(var/mob/user)
+	if(internal_battery && internal_battery.charge)
+		return 1
+	else ..()
+
+/obj/structure/bed/chair/vehicle/wheelchair/motorized/attackby(obj/item/weapon/W as obj, mob/user as mob)
+	if(isscrewdriver(W))
+		user.visible_message("<span class='notice'>[user] screws [maintenance ? "closed" : "open"] \the [src]'s battery compartment.</span>", "<span class='notice'>You screw [maintenance ? "closed" : "open"] the battery compartment.</span>", "You hear screws being loosened.")
+		maintenance = !maintenance
+	else if(iscrowbar(W)&&maintenance)
+		if(internal_battery)
+			user.put_in_hands(internal_battery)
+			internal_battery = null
+		user.visible_message("<span class='notice'>[user] pries out \the [src]'s battery.</span>", "<span class='notice'>You pry out \the [src]'s battery.</span>", "You hear a clunk.")
+	else if(istype(W,/obj/item/weapon/cell)&&maintenance&&!internal_battery)
+		user.drop_item(W,src)
+		internal_battery = W
+		user.visible_message("<span class='notice'>[user] inserts \the [W] into the \the [src].</span>", "<span class='notice'>You insert \the [W] into \the [src].</span>", "You hear something being slid into place.")
+	else ..()
+
+/obj/structure/bed/chair/vehicle/wheelchair/motorized/syndicate
+	nick = "medical malpractice"
+	desc = "A chair with fitted wheels which is powered by an internal cell. It seems to ride higher than other wheelchairs."
+	motor_delay = 0
```

Comment:
Wait, this means there's NO delay, doesn't it?

With the `Bump()` against borgs, I think that since there's no delay you'll basically instantly kill them.